### DISCUSSION TITLE
Mac OS expose privileged port error

### DISCUSSION THREAD

**Discussion by deimjons** [containers/podman#20258] [General]
Hi. I trying to migrate from Docker to Podman. When I run the container I receive an error: 

Error response from daemon: rootlessport cannot expose privileged port 80, you can add 'net.ipv4.ip_unprivileged_port_start=80' to /etc/sysctl.conf (currently 1024), or choose a larger port number (>= 1024): listen tcp 0.0.0.0:80: bind: permission denied

I have created file /etc/sysctl.conf and added there 'net.ipv4.ip_unprivileged_port_start=80 - but it didn't help. I tried to run the container with sudo and had the same error.  

How to fix it? Is it possible to fix it on Apple Silicon Mac?

**Comment by brianvess** [comment]
same problem here

**Comment by rhatdan** [comment]
'net.ipv4.ip_unprivileged_port_start=80' to /etc/sysctl.conf (currently 1024), or choose a larger port number (>= 1024): listen tcp 0.0.0.0:80: bind: permission denied

Did you do this within the MV and restart the VM?

**Reply by deimjons** [reply]
I do that on the host machine Mac OS in my case. I know that on mac doker/podman should use vm to run containers. I should do this setting in this vm? how to log in there? Should I ssh into the container and change settings there?

**Reply by rhatdan** [reply]
Yes.
